AcademyApp/ViewControllers/LecturesViewController.swift

```diff
@@ -81,6 +88,28 @@ extension LecturesViewController: UITableViewDelegate {
         let cellContentHeight = cellContentWidth * TableViewConfig.imageAspectRatio
         return cellContentHeight + TableViewConfig.cellVerticalInset
     }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        performSegue(withIdentifier: "goToLectureDetailViewController", sender: self)
```

Comment:
use enums for string constants

Reply:
This is only temporary solution. Navigation and VC initialization will be updated later on.